bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/types/BServiceType.java

```diff
@@ -32,14 +33,14 @@
  */
 public class BServiceType extends BObjectType implements ServiceType {
 
-    private ResourceFunctionType[] resourceFunctions;
-    private volatile MemberFunctionType[] remoteFunctions;
+    private ResourceMethodType[] resourceFunctions;
+    private volatile RemoteMethodType[] remoteFunctions;
 
     public BServiceType(String typeName, Module pkg, long flags) {
         super(typeName, pkg, flags);
     }
 
-    public void setResourceFunctions(ResourceFunctionType[] resourceFunctions) {
+    public void setResourceFunctions(ResourceMethodType[] resourceFunctions) {
```

Comment:
Shouldn't this be `setResourceMethods`?
```suggestion
    public void setResourceMethods(ResourceMethodType[] resourceMethods) {
```